contracts/HenkakuToken.sol

```diff
@@ -10,6 +10,7 @@ contract HenkakuToken is ERC20, Ownable{
     uint256 private maxSupply = 1_000_000_000e18; // 1 billion henkaku
     mapping(address => bool) private whitelist;
     address public gateKeeper;
```

Comment:
What is the intention behind the split between gatekeeper and dev? Are the permissions a little different?
It would be great if you could leave documentation in the code so we don't forget.

Reply:
ok, I assume gatekeeper would be multisig wallet which would be maintained by community
where as, dev would be the dev wallet address which add/update whitelist address behind like using crontab etc.